phpBB/styles/prosilver/template/forumlist_body.html

```diff
@@ -103,7 +103,11 @@
 								<!-- ENDIF -->
 								<br />{forumrow.LAST_POST_TIME}
 							<!-- ELSE -->
-								{L_NO_POSTS}<br />&nbsp;
+							{% if forumrow.U_UNAPPROVED_TOPICS %}
+								{{ lang('TOPIC_UNAPPROVED_FORUM', forumrow.TOPICS) }}<br />
```

Comment:
`<a href="{{ forumrow.U_UNAPPROVED_TOPICS }}">{{ lang('TOPIC_UNAPPROVED_FORUM', forumrow.TOPICS) }}</a><br />`

In case you want to directly link the authed user to the MCP without entering the forum.

Reply:
acctually we do not allow br tags anyway these will need to be factored out.

Reply:
This also makes changes to the templates which is a breaking change and should instead be pushed to 3.3 or 4.0

Reply:
This isn't a breaking change. If a style author does not apply this change it will not break their style. We routinely make style changes in 3.2 so this isn't any different.

Reply:
And unless we're going to remove all of the `<br />` tags in this file I'd just leave it there for this line. If there is a desire to change or remove these tags it should be done in a separate pr.

Reply:
Generally we like to prevent them if possilbe except for additions if absolutly required. Which I guess is what this is just did not look close enough ;)